### ISSUE TITLE
In this package whose account's client_id, client_secret, developer token, customer id and refresh token need to pass?

### ISSUE THREAD

**Issue by yashpatel521** [Opteo/google-ads-api#356]
> const client = new GoogleAdsApi({
>     client_id: '<CLIENT_ID>',
>     client_secret: '<CLIENT_SECRET>',
>     developer_token: '<DEVELOPER_TOKEN>',
> });

> const customer = client.Customer({
>     customer_account_id: '<CUSTOMER_ACCOUNT_ID>',
>     refresh_token: '<REFRESH_TOKEN>',
> });

**Comment by Zikoel** [comment]
All of that

**Comment by yashpatel521** [comment]
I have one google account ([yas...@gmail.com](https://groups.google.com/)) form which I have created a google ads account from one email address and second google account([yashM...@gmail.com](https://groups.google.com/)) in which I have created a manger account. I have one more account 
([yas...@gmail.com](https://groups.google.com/)) for manager account.

now from which account I have to get customer id, client Id ,client secret ,developer token and refresh token?

Do I have to link this account and if yes then how and which account?

My aim is to fetch the campaign list of (yashAds@gmail) through API.

provide email address are just examples.
